m4_sessions/1811_inning/confidence.md

```diff
@@ -1,18 +1,35 @@
 ## Group Members:
+Tara Craig, Patrick Shobe, Jesse Ling, Ben Ghalami, Becca Hyland
 
-
-
-## Name of Session: 
+## Name of Session: The Confidence Gap
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+(Why should students come to this session):
+
+Come explore current research on the relationship between confidence and competence (is there one?) and learn how to grow your own confidence and support others' confidence in our new careers as developers.
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+(What will students learn/leave with?):
+A toolbox: 
+1. Awareness of behavioral economic data / trends / research on the topic of confidence
+2. How to build confidence (strategies)
```

Comment:
What are the strategies?  Can you circle back to that at the end or highlight them specifically?